terraform/gcp/README.md

```diff
@@ -0,0 +1,34 @@
+# Terraform GCP
+Deploy Buz to Google Cloud via Terraform.
+
+1. Create a file in this directory `terraform.tfvars`. Set the `gcp_project` and optionally set `gcp_region`.
+```
+gcp_project = "my-project-23456"
+gcp_region  = "us-central1" #default value
+```
+
+2. In the command line authenticate GCP
+```
+gcloud auth application-default login
+```
+
+3. Create the artifact registry
+```
+terraform plan -target=google_artifact_registry_repository.buz_repository
+```
+
+4. After your registry has been created. Upload the appropriate buz image
+```
+gcloud auth configure-docker us-central1-docker.pkg.dev
+
+docker pull ghcr.io/silverton-io/buz:v0.11.11@sha256:130ed9421579125e4f38089e4c2d1e07038fb26591a15082010a52b95f3a5dda # amd64
```

Comment:
Referencing specific shas like this seems a bit brittle.  it's already outdated... we are on:
buz:v0.11.11@sha256:cd285efbb43b17a021f236bb846aef6f07ada1a54b4bc313843a73cd8b884309

Reply:
I wrote that here: https://github.com/silverton-io/buz/pull/397

The issue is cloud run currently doesn't run arm-based images. The alternative is passing a `--platform=linux/amd64` flag (which is prob the way).